UKHomeOffice/nmsw-ui
# NMSW-1308 Change to declaration endpoint UPT page

# Ticket



----

## AC
We're moving to
`/declaration/<id>` for declaration data
`/declaration/<id>/attachments` for when you need the attachments

FE needs updating to match this

This ticket covers `VoyageUPTStatusResults`


----

## To test

check
- unit tests pass
- you can still upload a FAL6
- you can view the UPT request page for that FAL6
- you can go back to dashboard and return to this declaration and view the UPT request page
- you can upload a replacement FAL6
- you can submit this declaration

----

## Notes
